CMakePresets.json

```diff
@@ -1,5 +1,4 @@
 {
-    "$schema": "https://cmake.org/cmake/help/latest/_downloads/3e2d73bff478d88a7de0de736ba5e361/schema.json",
```

Comment:
Right, CMake-Tools is supporting CMakePresets.json version 6, but the $schema attribute was introduced in version 8.